apps/cowswap-frontend/src/modules/application/containers/App/index.tsx

```diff
@@ -49,6 +52,8 @@ export function App() {
   useAnalyticsReporterCowSwap()
   useInitializeUtm()
 
+  const [featureFlags] = useAtom(featureFlagsAtom)
```

Comment:
`useFeatureFlags()` can be used here in order to keep it simple

Reply:
Addressed